kf_lib_data_ingest/common/warehousing.py

```diff
@@ -1,6 +1,9 @@
 from urllib.parse import urlparse
 
 import sqlalchemy
+from sqlalchemy.schema import DropSchema
+
+SCHEMA_PREFIX = "ingest:"
```

Comment:
Nitpick, but can we make it `Ingest` since the rest of the schema will be camel cased. Then it'll look like `Ingest:ExtractStage`

Reply:
🤷‍♂  ok